### ISSUE TITLE
Cannot insert Emojis when text entered.

### ISSUE THREAD

**Issue by wilriker** [threema-ch/threema-web#256]
## Expected Behavior
When I enter text into the text field and then try to add an Emoji I expect the Emoji to appear in the current position of the cursor.

## Current Behavior
No Emoji is inserted (it works when no text is entered) and the console shows the following error message:
```
app.js?v=1.3.0:10052 Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at HTMLSpanElement.onEmojiChosen (app.js?v=1.3.0:10052)
    at defaultHandlerWrapper (angular.js?v=1.3.0:3546)
    at HTMLSpanElement.eventHandler (angular.js?v=1.3.0:3534)
onEmojiChosen @ app.js?v=1.3.0:10052
defaultHandlerWrapper @ angular.js?v=1.3.0:3546
eventHandler @ angular.js?v=1.3.0:3534
```

This issue also happens in incognito mode of the browser with all extensions disabled.

## Possible Solution
-/-

## Steps to Reproduce (for bugs)
1. Enter text
2. Try to add an emoji

## Your Environment
* Threema Web version: 1.3.0
* Browser name and version: Google Chrome Version 59.0.3071.109 (Official Build) (64-bit)
* Operating system and version: CentOS 7.3.1611

## Additional Information
It does not happen in Firefox ESR 52.2.0 (64-bit) (freshly installed for the purpose of testing this issue).

**Comment by dbrgn** [comment]
Can confirm. It only happens in Chrome and under certain circumstances.

**Comment by Kurisutian** [comment]
It also happens in Rambox

**Comment by dbrgn** [comment]
Should be fixed in 1.3.1.

**Comment by wilriker** [comment]
Fix confirmed!
